src/components/data-model-table/index.tsx

```diff
@@ -19,6 +19,8 @@ export interface IDataModelListProps {
   hideCreateButton?: boolean;
   hideActions?: boolean;
   parentView?: ResourceType;
+  height?: string;
```

Comment:
I dont think we should need the height or the viewMode props. The height should be 100%, the map should grow and take up whatever size the parent container makes available. The viewMode I think for now is also not needed as a PROP because the LocationList for now will always render the table first... but I suppose it wouldnt hurt too much to leave this one in in cases where we want to show LocationList with the map as default... But the height, ideally, it would be helpful if it just automatically figured it out via css and was responsive to fill the parent container